src/amo/components/Categories/index.js

```diff
@@ -51,6 +51,12 @@ export class CategoriesBase extends React.Component {
     const { addonType, className, error, loading, i18n } = this.props;
     const categories = this.props.categories[addonType] ?
       Object.values(this.props.categories[addonType]) : [];
+    categories.sort(function(a,b){
```

Comment:
There's nothing wrong with this style of function declaration but we usually use fat-arrow functions:

```js
categories.sort((a, b) => {
  [...]
});
```